src/main/java/graphql/schema/DataFetcher.java

```diff
@@ -3,5 +3,5 @@
 
 public interface DataFetcher {
```

Comment:
I'm having trouble wrapping my mind around this :). Wouldn't this have some pretty far reaching consequences?